lnbits/static/i18n/pt_PT.js

```diff
@@ -0,0 +1,88 @@
+  export_to_phone: 'Exportar para o telefone com código QR',
+  export_to_phone_desc:
+    'Este código QR contém a URL da sua carteira com acesso total. Você pode escaneá-lo do seu telefone para abrir sua carteira a partir dele.',
+  wallets: 'Carteiras',
+  add_wallet: 'Adicionar nova carteira',
+  delete_wallet: 'Excluir carteira',
+  delete_wallet_desc:
+    'Toda a carteira será excluída, os fundos serão IRRECUPERÁVEIS.',
+  rename_wallet: 'Renomear carteira',
+  update_name: 'Atualizar nome',
+  press_to_claim: 'Pressione para solicitar bitcoin',
+  donate: 'Doar',
+  view_github: 'Ver no GitHub',
+  voidwallet_active: 'VoidWallet está ativo! Pagamentos desabilitados',
+  use_with_caution: 'USE COM CAUTELA - a carteira %{name} ainda está em BETA',
+  toggle_darkmode: 'Alternar modo escuro',
+  view_swagger_docs: 'Ver a documentação da API do LNbits Swagger',
+  api_docs: 'Documentação da API',
+  commit_version: 'Versão de commit',
+  lnbits_version: 'Versão do LNbits',
+  runs_on: 'Executa em',
+  credit_hint: 'Pressione Enter para creditar a conta',
+  credit_label: '%{denomination} para creditar',
+  paste_request: 'Colar Pedido',
+  create_invoice: 'Criar Fatura',
+  camera_tooltip: 'Usar a câmara para escanear uma fatura / QR',
+  export_csv: 'Exportar para CSV',
+  transactions: 'Transações',
+  chart_tooltip: 'Mostrar gráfico',
+  pending: 'Pendente',
+  copy_invoice: 'Copiar fatura',
+  close: 'Fechar',
+  cancel: 'Cancelar',
+  scan: 'Escanear',
+  read: 'Ler',
+  pay: 'Pagar',
+  memo: 'Memo',
+  date: 'Data',
+  processing_payment: 'Processando pagamento...',
+  not_enough_funds: 'Fundos insuficientes!',
+  search_by_tag_memo_amount: 'Pesquisar por tag, memo, quantidade',
+  invoice_waiting: 'Fatura aguardando pagamento',
+  payment_received: 'Pagamento Recebido',
+  payment_sent: 'Pagamento Enviado',
+  outgoing_payment_pending: 'Pagamento pendente de saída',
+  drain_funds: 'Drenar Fundos',
+  drain_funds_desc:
+    'Este é um código QR de retirada do LNURL para sugar tudo desta carteira. Não compartilhe com ninguém. É compatível com balanceCheck e balanceNotify para que sua carteira possa continuar retirando os fundos continuamente daqui após a primeira retirada.',
```

Comment:
I would translate lnurl withdrawl to "saque lnurl"
Este é um código QR de saque LNURL para sacar tudo desta carteira. Não o partilhe com ninguém. É compatível com balanceCheck e balanceNotify para que a sua carteira possa continuar levantando os fundos continuamente daqui após o primeiro saque.